unipy_logger/uni_logger.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/python3
+
+import time
+import datetime
+import os
+import sys
+import json
+from logbook import Logger, StreamHandler
+
+
+CRITICAL = "CRITICAL"
+ERROR = "ERROR"
+WARNING = "WARNING"
+NOTICE = "NOTICE"
+INFO = "INFO"
+DEBUG = "DEBUG"
+
+LOG_LEVELS = {
+    CRITICAL: 2,
+    ERROR: 3,
+    WARNING: 4,
+    NOTICE: 5,
+    INFO: 6,
+    DEBUG: 7
+}
+
+
+class UniLogger:
+    __instance = None
+    logger = None
+    context = {}
+    messageLevel = ""
+
+    def __init__(self, stream, level):
+        """ Virtually private constructor. """
+        if UniLogger.__instance != None:
+            raise Exception("Logger is already been instantiated")
```

Comment:
"Logger is already instantiated" or "Logger has already been instantiated" 🇬🇧

Reply:
https://memegenerator.net/img/instances/500x/52806285/watch-out-we-gotta-grammar-nazi-over-here.jpg

Reply:
The Queen would be proud. 🍵  👑